src/util/corePlugins.js

```diff
@@ -80,6 +80,7 @@ export const swapPlugins = {
   foxExchange: ENV.FOX_INIT,
   godex: ENV.GODEX_INIT,
   // shapeshift: ENV.SHAPESHIFT_INIT,
+  switchain: ENV.SWITCHAIN_INIT,
```

Comment:
Can you add your block to env.example.json?

Reply:
Added